tensorflow/tools/docker/Dockerfile.devel-rocm

```diff
@@ -205,7 +135,7 @@ RUN mkdir /bazel && \
 
 # Download and build TensorFlow.
 WORKDIR /tensorflow-upstream
-RUN git clone --branch=.* --depth=1 https://github.com/ROCmSoftwarePlatform/tensorflow-upstream.git .
+RUN git clone --branch=r1.10-rocm --depth=1 https://github.com/ROCmSoftwarePlatform/tensorflow-upstream.git .
```

Comment:
@parallelo any reason we stick with `r1.10-rocm` branch here, instead of `develop-upstream`?

Reply:
That's a good question.  The other Dockerfiles (e.g. `Dockerfile.devel-gpu`) are already pointing to r1.11, so I just used the most recent TF release for ROCm.  I'm open to changing this though.

Reply:
could you help clarify the purpose of the dockerfile? if it's deemed to stick with a particular released version, it's fine to use `r1.10-rocm`. or if it's always the cutting edge tip we need, maybe we should use `develop-upstream`.

Reply:
To check against ROCm staging bits, we should use a TF release branch.

Reply:
ok i see. then `r1.10-rocm` it is.